CMakeLists.txt

```diff
@@ -51,13 +49,13 @@ if(MDNSD_BUILD_FUZZING OR MDNSD_BUILD_OSS_FUZZ)
     add_subdirectory(tests/fuzz)
 endif()
 
-add_executable(mdnsd mdnsd.c)
-target_link_libraries(mdnsd PRIVATE libmdnsd)
+add_executable(mdns_daemon mdnsd.c)
+target_link_libraries(mdns_daemon PRIVATE mdnsd)
```

Comment:
A mdns.exe execuable cannot have a conflict with a mdns.lib library.

I opened https://github.com/Pro/mdnsd/pull/23 without seeing your pr,
and there I am not seeing this problem.